src/main/java/com/imagelab/App.java

```diff
@@ -0,0 +1,39 @@
+package com.imagelab;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+/**
+ * JavaFX App
+ */
+public class App extends Application {
+
+    private static Scene scene;
+
+    @Override
+    public void start(Stage stage) throws IOException {
+        scene = new Scene(loadFXML("dashboard"), 1285, 785);
+        stage.setScene(scene);
+        stage.setResizable(false);
+        stage.show();
+    }
+
+    static void setRoot(String fxml) throws IOException {
+        scene.setRoot(loadFXML(fxml));
+    }
+
+    private static Parent loadFXML(String fxml) throws IOException {
```

Comment:
Let's use doc comments for any method except an overridden one.

Reply:
Sure I will add them